quality/integration/src/test/resources/cleanup_tables.sh

```diff
@@ -34,13 +34,12 @@ function main() {
   fi
 
   api_key_header="X-BV-API-Key: ${api_key}"
-  tables=$(curl -H "${api_key_header}" "${url}/sor/1/_table?limit=2147483647" | jq .[].name)
+  tables=$(curl -s -H "${api_key_header}" "${url}/sor/1/_table?limit=2147483647" | jq .[].name)
 
-  # Go through each prefix
-  for table_prefix in ${table_prefixes[*]}; do
-    log "Deleting tables for $table_prefix prefix"
+  log "Deleting tables for $table_prefix prefix"
     # And get the tables that match the prefix
-    for table in $(echo "$tables"| grep "${table_prefix}" | tr -d '"'); do
+    filtered_tables=$(echo "${tables}" | tr -d '"' | grep "^${table_prefix}*")
```

Comment:
to iterate over tables that only start with `table_prefix` more efficient solution would be `for table in $(echo "$tables"| grep "^${table_prefix}" | tr -d '"'); do`

Reply:
Okay, with only tine difference, otherwise it will not work 
it should be `for table in $(echo "$tables"| tr -d '"' | grep "^${table_prefix}"); do`

Reply:
As discussed, in order to decrease number of data formatting activities it can be done this way `$(echo "${tables}" | grep "^\\\"${table_prefix}" | tr -d '"' )`